utils/api/fetchApy.ts

```diff
@@ -1,19 +1,29 @@
 async function fetchApy() {
-  const apyDayOptions = [7, 30, 365];
-  const dayResults = await Promise.all(
-    apyDayOptions.map(async (days) => {
-      const endpoint = `${process.env.NEXT_PUBLIC_ANALYTICS_ENDPOINT}/api/v2/oeth/apr/trailing/${days}`;
-      const response = await fetch(endpoint);
-      if (!response.ok) {
-        throw new Error(`Failed to fetch ${days} day APY`);
-      }
-      const json = await response.json();
-      return json.apy / 100;
-    })
-  ).catch(function (err) {
-    console.log(err.message);
-  });
-  return dayResults;
+  try {
+    const res = await fetch(process.env.NEXT_PUBLIC_SUBSQUID_URL, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        query: `query APY {
+          apies(orderBy: blockNumber_DESC, limit: 1) {
+            apy7DayAvg
+            apy30DayAvg
+            id
+          }
+        }`,
+        variables: null,
+        operationName: "APY",
+      }),
+    });
+    const json = await res.json();
+
+    return [
+      json.data.apies[0].apy7DayAvg / 100,
+      json.data.apies[0].apy30DayAvg / 100,
```

Comment:
Sad to say it but the resolver should be returning these values in the correct percentages.  We should not have to do this. The data squid gives us is bad. We should fix that.

Reply:
7.5321496604432046 -> 0.075321496604432046

Reply:
Since it is a resolver change (I think) we should be able to maintain v6 I'm guessing.

Reply:
nevermind, not your fault